big-int/src/bigint.hpp

```diff
@@ -125,7 +125,7 @@ class largeInt
     bool operator!=(int);
     bool operator!=(int64_t);
 
-    bool operator<(largeInt);
+    bool operator<(const largeInt& z);
```

Comment:
no need of specifying argument name in function declaration, let it be `bool operator < (const largeInt&);`